docs/sql/statements/insert.rst

```diff
@@ -42,6 +43,23 @@ filled.
 If the expression for any column is not of the correct data type, automatic
 type conversion will be attempted.
 
+The optional RETURNING clause causes INSERT to compute and return value(s)
+based from each row actually inserted (or updated, if an ON CONFLICT DO UPDATE
+clause was used). This is primarily useful for obtaining values that were
+supplied by defaults, such as a :ref:`sequence number
+<sql_administration_system_columns_seq_no>`.
```

Comment:
Isn't `seq_no` quite advanced already? I think the most popular and simple use-case is to return generated primary keys, e.g. `_id` or  `md5(random()::text || current_timestamp::text)`

Reply:
Sequence number was used in the original psql documentation and it is also one of our usecases, but we could go indeed for something simpler.

Reply:
No strong opinion, just an idea.
Btw. postgresql documentation talks about a serial sequence number in general. Also just to mention, our `_seq_no` is only serial per shard.

Reply:
I like primary key